userspace/falco/app_actions/open_inspector.cpp

```diff
@@ -22,13 +22,15 @@ limitations under the License.
 
 #include "application.h"
 
+#define FALCO_BPF_ENV_VARIABLE "FALCO_BPF_PROBE"
```

Comment:
TODO: should we drop anything in the cmake files?

Reply:
Yep, we needed to drop also `SCAP_BPF_PROBE_ENV_VAR_NAME` cmake variable, as it was unsued now.
Dropped in latest commit.